apps/communications/dialer/test/marionette/lib/views/call_log/accessors.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
```

Comment:
no problem, this was created just for testing/practicing. It was easy to change from one tab to another that going to another app. I can remove it